src/details/ArborX_DetailsBatchedQueries.hpp

```diff
@@ -125,7 +125,7 @@ struct BatchedQueries
           tmp_offset(permute(i)) = offset(i + 1) - offset(i);
         });
 
-    exclusivePrefixSum(tmp_offset);
+    exclusivePrefixSum(ExecutionSpace{}, tmp_offset);
```

Comment:
Is there a plan for utility functions to also take a space in the future? If not, why?

Reply:
Yes.  All the way to `BVH::query()`